### PR TITLE
Changed code header

### PR DESCRIPTION
**Pull request by jeremiah**
SPDX-ification.

### CODE REVIEW

File: plugin.cpp [hunk 1]
```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (C) 2016, Pelagicore AB.
+ * Author: Jonatan Pålsson <jonatan.palsson@pelagicore.com>
+ * SPDX-License-Identify: MPL-2.0
+ */
```
Comment (breakreturn) [inline plugin.cpp:5]: This doesn't really look like the template example we have does it? https://github.com/Pelagicore/OpenSourceTemplates/blob/master/examples/Example_header_cpp_mpl.md

File: plugin.cpp [hunk 2]
```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (C) 2016, Pelagicore AB.
+ * Author: Jonatan Pålsson <jonatan.palsson@pelagicore.com>
+ * SPDX-License-Identify: MPL-2.0
```
Comment (breakreturn) [inline plugin.cpp:4]: Should this be "SPDX-License-Identifier" ?

**Review by breakreturn — CHANGES_REQUESTED** [review]

**Review by jonte — CHANGES_REQUESTED** [review]
Hi Jeremiah,

Thanks for the patch.

I would prefer to format the file headers in the following way: https://github.com/Pelagicore/bluetooth-demo/blob/master/bluephonebook.cpp#L6

As in, include the short version of the license text, and leave the SPDX identifier out (the license analysys tools select the correct license anyway based on the license header).

Will you change your patch?

File: plugin.cpp (hunk 1 above)
Comment (jonte) [reply plugin.cpp:5]: Ah. I wasn't aware of this example file when I wrote my previous comment. If there is a standard within the company, then we should follow it. I would argue that this standard can be improved though (same arguments as my previous comment).

Comment (breakreturn) [reply plugin.cpp:5]: @jonte The templates are there to be the "current truth" so to speak. I.e. they enable us to not have the same discussions every time we create a new code file, but they are also intended to be improved over time when anyone have good input.

Comment (jonte) [reply plugin.cpp:5]: I will open a discussion on the template file on GitHub, and we can discuss how it should look in that thread.

**Review by DunderRoffe — APPROVED** [review]
Given that the concerns of @breakreturn and @jonte are addressed I approve this change.

**Comment by jeremiah** [comment]
I'm going to close this PR. I'll create a new PR that follows the policy we agreed to and that is published in the Software Factory documentation.

**Pull request closed without merging**